frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -292,9 +294,28 @@ void retry() {
             return;
         }
 
-        logger.info("Retry worker: {}", workerId);
-        ModelManager modelManager = ModelManager.getInstance();
-        modelManager.submitTask(this);
+        running.set(false);
+        ModelManager manager = ModelManager.getInstance();
+
+        backoffIdx = (backoffIdx + 1) < Model.BACKOFF.length ? backoffIdx + 1 : backoffIdx;
```

Comment:
backoffIdx = (backofIdx + 1) % Model.BACKOFF.length;

Reply:
I thought of percentile but thought the ternary operator was more explicit. Are you suggesting this change for performance or for code elegance?